src/style.css

```diff
@@ -1 +1,66 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf) format('truetype');
+  font-weight: 300;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf) format('truetype');
+  font-weight: 900;
+  font-style: normal;
+}
+
+.search__form {
+  margin-top: 20px;
+}
+
+.search__input {
+  font-family: Avenir, Arial, Helvetica, sans-serif, Verdana;
```

Comment:
According to the checklist, you should have a fallback font in case the main one doesn't work. It's recommended to add a generic font family at the end of your font stack, like 'sans-serif' or 'serif'. This way, if the browser can't load the specified font for some reason, it will default to a generic font that should be available on all systems.